app/views/shared/_webchat.html.erb

```diff
@@ -2,8 +2,9 @@
 
 <p>
   <span class="js-webchat"
-    data-availability-url="<%= @content_item.webchat_availability_url %>"
-    data-open-url="<%= @content_item.webchat_open_url %>">
+    data-availability-url="<%= @content_item.webchat.availability_url %>"
+    data-open-url="<%= @content_item.webchat.open_url %>"
```

Comment:
This is great! Could the lines about looping through the config be removed from the commit message too?